src/main/java/org/sagebionetworks/bridge/user/TestUserHelper.java

```diff
@@ -132,10 +132,23 @@ public void setClientInfo(ClientInfo clientInfo) {
                     .withAcceptLanguage(LANGUAGES).build();
         }
     }
+    /**
+     * Get the signed in, bootstrap admin user. This method will reset the administrator's study 
+     * to be the API/test study, because this is a precondition expected by most of our tests and 
+     * it's easy to break by failing to reset the admin users's study as part of test cleanup. 
+     */
     public static TestUser getSignedInAdmin() {
         if (cachedAdmin == null) {
             cachedAdmin = getSignedInUser(CONFIG.getAdminSignIn());
         }
+        String testStudyId = CONFIG.fromProperty(Config.Props.API_DEVELOPER_STUDY);
```

Comment:
API_DEVELOPER_STUDY is going to be removed, as per https://sagebionetworks.jira.com/browse/BRIDGE-2086

The canonical config for the API study is Props.STUDY_IDENTIFIER